site/hxapi/de-start/bkg.md

```diff
@@ -0,0 +1,170 @@
+---
+
+---
+
+# Book Car Park - DE-Start method
+
+## Car Park Booking Request
+
+### Method
+
+POST
+
+### Endpoint
+
+The endpoint to use is:
+
+```
+https://payment.holidayextras.co.uk/legacy/carpark/CarParkCode
```

Comment:
Is this correct? The hostname `payment.holidayextras.co.uk` does no appear to resolve.